Assets/Scripts/System/GameSync.cs

```diff
@@ -57,7 +57,7 @@ void Update()
     /// </summary>
     public void execute()
     {
-        if (syncType != SyncType.NONE)
+        if (true) // syncType != SyncType.NONE)
```

Comment:
This is buggy I think. Sync was never running. I'll come back and investigate, but for the moment I thought best to always sync rather than never. 😅